src/org/parosproxy/paros/view/View.java

```diff
@@ -168,6 +175,8 @@
     // ZAP: splash screen
     private SplashScreen splashScreen = null;
 
+    private static Map<AddOn.Status, StatusUI> statusMap = new HashMap<AddOn.Status, StatusUI>();

```

Comment:
Would you mind changing the statusMap to instance variable and the method getStatusUI(...) to instance method? I do think that's better in the long run.